<?xml version="1.0" ?>
<launch>

  <arg name="omni" default="false" />
  <arg name="parameter_index_irn" default="0" />
  <arg name="parameter_index_utn" default="1" />

  <!-- Reconfigure -->
  <node pkg="rasberry_optimise" type="reconf_at_edges.py" name="reconf_at_edges" output="screen" 
        args="$(arg omni) $(arg parameter_index_irn) $(arg parameter_index_utn)" />

</launch>
